docs/concepts/vega-chain/network.md

```diff
@@ -1,27 +1,178 @@
+| **Proof of work parameters** | |
+| spam.pow.numberOfTxPerBlock       	| Maximum number of transactions a wallet can put in one block 	| <NetworkParameter frontMatter={frontMatter} param="spam.pow.numberOfTxPerBlock" hideName={true} />
+| spam.pow.difficulty               	| To prevent flooding the chain with transactions, wallets need to perform a proof-of-work (PoW) calculation to submit them. This defines the difficulty level	| <NetworkParameter frontMatter={frontMatter} param="spam.pow.difficulty" hideName={true} />
+| spam.pow.hashFunction              	| Hash function used for proof-of-work	| <NetworkParameter frontMatter={frontMatter} param="spam.pow.hashFunction" hideName={true} />
+| spam.pow.increaseDifficulty           | If a wallet exceeds the max transactions per block, the difficulty of the PoW increases by this factor | <NetworkParameter frontMatter={frontMatter} param="spam.pow.increaseDifficulty" hideName={true} />
+| spam.pow.numberOfPastBlocks         	| To compute the wallet transaction quota, transactions can be assigned to past blocks; this parameter defines how far back that goes | <NetworkParameter frontMatter={frontMatter} param="spam.pow.numberOfPastBlocks" hideName={true} />
+
+## Network restarts: Checkpoints
 The network's validators periodically store checkpoints of all important state parameters such as balances and governance proposals. 
 
-Checkpoints allow the chain to be restarted from a previously valid state in the event of consensus failure, or a critical issue being discovered. 
+Checkpoints allow the chain to be restarted from a previously valid state in the event of consensus failure, a network restart, or a critical issue being discovered.
 
 Those checkpoints happen at defined intervals, and on every deposit and withdrawal request. 
 
-1. Each validator node calculates the hash of the checkpoint file and then sends this through consensus to ensure all the nodes in the network agree on the state. 
-2. If/when a network is taken down, the checkpoint file's hash is added to the genesis block. 
+### How a checkpoint is created and used
+1. Each validator node calculates the hash of the checkpoint file and then sends this through consensus to ensure all the nodes in the network agree on the state. A checkpoint file is then automatically produced and saved with a hash. 
+2. When a network is taken down, the checkpoint file's hash is added to the genesis block. A node operator updates the genesis file in the networks repo with the network's new start date, the new network ID, and the hash of the checkpoint file to be used in the network restoration. Once the file is updated, the node operators manually approve the change, and then one validator submits the restore transaction with the relevant checkpoint file.
```

Comment:
this is no longer true. There is no checkpoint restore command. The checkpoint data is embedded in the genesis file.